Nos/Controller/NoteEditorController.swift

```diff
@@ -2,23 +2,23 @@ import Foundation
 import SwiftUI
 import UIKit
 
-/// A controller for Nostr note text that is being edited. This controllers pairs with a `NoteUITextViewRepresentable`
+/// A controller for Nostr note text that is being edited. This controller pairs with a ``NoteUITextViewRepresentable``
```

Comment:
When I do Product > Build Documentation, which Xcode prompted me to do, I get warnings on this and all the other NoteUITextViewRepresentable references. The NoteTextEditor reference below works perfectly and links to it (🤯) but the NoteUITextViewRepresentable references don't.

<img width="1000" alt="Screenshot 2024-08-22 at 10 22 09 AM" src="https://github.com/user-attachments/assets/d2d305a5-9ad8-4c73-adf6-d68a76be2c33">

Update: I think the issue is that NoteUITextViewRepresentable needs to be NoteTextEditor/NoteUITextViewRepresentable, and everything seems good for me.

Reply:
Thanks, yeah I noticed that too. Strange. I'll check it out.

Reply:
I decided to remove the double backticks because I couldn't make sense of this warning, and it didn't seem to warrant more time right now. I've never seen this issue before, even in a much bigger Xcode project. Can revisit another time.

Reply:
oh, whoops, I missed this comment and just suggested we add it again. I think it'll be ok if we do what I suggested below